variants/aws-dev/Cargo.toml

```diff
@@ -47,6 +48,12 @@ path = "../variants.rs"
 [build-dependencies]
 # core
 early-boot-config = { path = "../../packages/early-boot-config" }
+early-boot-config-ec2-identity-doc = { path = "../../packages/early-boot-config-ec2-identity-doc" }
+early-boot-config-ec2-imds = { path = "../../packages/early-boot-config-ec2-imds" }
+early-boot-config-local-defaults = { path = "../../packages/early-boot-config-local-defaults" }
+early-boot-config-local-overrides = { path = "../../packages/early-boot-config-local-overrides" }
+early-boot-config-local = { path = "../../packages/early-boot-config-local" }
+early-boot-config-providers = { path = "../../packages/early-boot-config-providers" }
```

Comment:
I'm a bit surprised by this. Couldn't these just refer to the appropriate metapackage?